examples/view-exns/index.txt

```diff
@@ -10,9 +10,3 @@ Verifying for generic connector
   Verifying when ALL participants are honest
   Verifying when NO participants are honest
 Checked 5 theorems; No failures!
-Compiling "main"...
```

Comment:
Why did this get removed?

Reply:
`main = dapp_plus` which was already compiled, but we probably want to re-write the output files as `main` too